samples/java_layout/src/main/java/io/customer/android/sample/java_layout/data/model/CustomerIOSDKConfig.java

```diff
@@ -98,22 +119,29 @@ public static Map<String, String> toMap(@NonNull CustomerIOSDKConfig config) {
     private final Integer flushInterval;
     @Nullable
     private final Integer flushAt;
-    @Nullable
-    private final Boolean screenTrackingEnabled;
-    @Nullable
-    private final Boolean deviceAttributesTrackingEnabled;
-    @Nullable
-    private final Boolean debugModeEnabled;
+    private final boolean screenTrackingEnabled;
```

Comment:
It wasn't clear to me that it was intentional so I just wanted to keep it consistent with the new values. Do we still want to keep them as nullable values? Having two getters for each field returning nullable boolean and primitive boolean was too confusing.

Do we still want to differentiate between default values and user values? Not sure if we actually use that distinction anywhere though

Reply:
I agree that using object classes adds some management overhead, but primitive types will always resolve to either `true` or `false` which can't fully capture the essence of SDK default values. Although it only becomes relevant if SDK's default value changes. I can't think of any other major use cases where it would matter, so as long as saving and restoring values features are working fine, I think using primitives is fine and won't really make a difference in most cases.